core/extensions/extension_factory.hpp

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_EXTENSIONS_EXTENSION_FACTORY_HPP
+#define KAGOME_CORE_EXTENSIONS_EXTENSION_FACTORY_HPP
+
+#include "extensions/extension.hpp"
+
+namespace kagome::extensions {
+
+  class ExtensionFactory {
```

Comment:
Please comment.

Reply:
I wanted to comment this, but ended up with comments like: Factory for extensions. 
And for method: creates extension.
If you think this is reasonable comments then I can add them